console-subscriber/src/lib.rs

```diff
@@ -349,3 +388,26 @@ impl Visit for FieldVisitor {
         });
     }
 }
+
+impl Visit for WakerVisitor {
+    fn record_debug(&mut self, _: &field::Field, _: &dyn std::fmt::Debug) {
+        // don't care (yet?)
+    }
+
+    fn record_u64(&mut self, field: &tracing_core::Field, value: u64) {
+        if field.name() == "task.id" {
+            self.id = Some(span::Id::from_u64(value));
+        }
+    }
+    fn record_str(&mut self, field: &tracing_core::Field, value: &str) {
```

Comment:
nit: missing whitespace here
```suggestion
    }
    
    fn record_str(&mut self, field: &tracing_core::Field, value: &str) {
```